run_ac.py

```diff
@@ -55,6 +55,8 @@ def __check_data_type_text(attr_value):
         return False
     return True
 
+def __print_progress(progress: float) -> None:
+    print(f":progress:{progress}", flush=True)
```

Comment:
might be wrong but i think there is a : to much in the beginning.

Reply:
I changed it to this so I could just split with ":" as a seperator and then get the progress with values[-1] if values[-2] == "progress"

Reply:
not the case anymore the code in gateway reads 
`if ":progress:" in log:`

Reply:
Yes I can remove the first ":", I think it does not cause any problems but it does not benefit and therefore can be removed